cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java

```diff
@@ -226,6 +225,9 @@ private long calculateUpdateInterval() {
    * @return The update interval in milliseconds.
    */
   public long getUpdateInterval() {
+    if (updateInterval == null) {
```

Comment:
sometimes, we don't schedule it (so we dont call getUpdateInterval), but simply call its update method (such as during launching of programs).
So, when launching 100 workflows, it will no longer log 100x the log message at the bottom of computeUpdateInterval(), which can be misleading.